hack/tools/go.sum

```diff
@@ -613,8 +613,8 @@ rsc.io/quote/v3 v3.1.0/go.mod h1:yEA65RcK8LyAZtP9Kv3t0HmxON59tX3rD+tICJqUlj0=
 rsc.io/sampler v1.3.0/go.mod h1:T1hPZKmBbMNahiBKFy5HrXp6adAjACjK9JXDnKaTXpA=
 sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230510161459-7571f4b9a942 h1:hFD347lSM3XkuHzQYmZzna9ibtrgaw8qGNPI3eEsrVo=
 sigs.k8s.io/controller-runtime/tools/setup-envtest v0.0.0-20230510161459-7571f4b9a942/go.mod h1:B6HLcvOy2S1qq2eWOFm9xepiKPMIc8Z9OXSPsnUDaR4=
-sigs.k8s.io/controller-tools v0.13.0 h1:NfrvuZ4bxyolhDBt/rCZhDnx3M2hzlhgo5n3Iv2RykI=
```

Comment:
This is a downgrade, I wonder if this is ok like this.

Reply:
Okay now I see it is actually un-buildable, should we rebase or cancel this PR?

Reply:
I am not sure how to ask Dependabot to recreate the PR.

Reply:
Well we can cancel it or rebase it, re-creating I am not sure.